_data/tools.json

```diff
@@ -165,7 +165,7 @@
                 "material": [
                     {
                         "name": "Notebook",
-                        "link": "https://binder.bioimagearchive.org/v2/gh/ome/omero-guide-ilastik/v0.2.3?filepath=notebooks/idr0062_pixel_classification_zarr.ipynb",
+                        "link": "https://binder.bioimagearchive.org/v2/gh/ome/omero-guide-ilastik/v0.3.0?filepath=notebooks/idr0062_pixel_classification_zarr.ipynb",
```

Comment:
For me this failed to build with 

```
zstandard                 0.15.2                   pypi_0    pypi
zstd                      1.4.5                h9ceee32_0    anaconda
Removing intermediate container 97165c03c270
Error processing tar file(exit status 1): open /srv/conda/envs/notebook/lib/python3.7/site-packages/numcodecs/lz4.cpython-37m-x86_64-linux-gnu.so: cannot allocate memory
```

Reply:
some issue on binder.bioimagearchive.org.
Migration to v4 is ongoing